pkgs/tools/misc/nginx-config-formatter/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub, python3 }:
+
+stdenv.mkDerivation rec {
+  version = "2016-06-16";
+  name = "nginx-config-formatter-${version}";
```

Comment:
@rycee you're right
https://nixos.org/nix/manual/#versions
I thought Nix didn't version comparisons but it does.